apps/angular/projection/src/app/ui/card/card.component.ts

```diff
@@ -37,25 +32,18 @@ import { ListItemComponent } from '../list-item/list-item.component';
     </div>
   `,
   standalone: true,
-  imports: [NgIf, NgFor, ListItemComponent],
+  imports: [ListItemComponent, NgTemplateOutlet],
 })
 export class CardComponent {
```

Comment:
use a generic, it's nicer than using any

Reply:
done